### PR TITLE
add sriov topology manager doc

### PR DESCRIPTION
**Pull request by lamek**

### CODE REVIEW

File: modules/nw-sriov-topology-manager.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+oc describe pod sample-pod
+----
+
+. Optional: Confirm that `sample-pod` is allocated with exclusive CPUs.
++
+----
+oc exec sample-pod -- cat /sys/fs/cgroup/cpuset/cpuset.cpus
+----
+
+. Optional: Confirm that SR-IOV device and CPUs allocated for `sample-pod` are on the same NUMA node.
```
Comment (jboxman) [inline modules/nw-sriov-topology-manager.adoc]: s/that/that the/
s/allocated/that are allocated/
s/for the/

File: modules/nw-sriov-topology-manager.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+----
+<1>  Replace `<filename>` with the name of the file you created in the previous step.
+
+. Optional: Confirm that `sample-pod` is configured with guaranteed QoS.
++
+----
+oc describe pod sample-pod
+----
+
+. Optional: Confirm that `sample-pod` is allocated with exclusive CPUs.
```
Comment (jboxman) [inline modules/nw-sriov-topology-manager.adoc]: s/that/that the/

File: modules/nw-sriov-topology-manager.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="nw-sriov-topology-manager_{context}"]
+= Creating NUMA aligned SR-IOV pod
+
+You can create a NUMA aligned SR-IOV pod by restricting SR-IOV and the CPU resources allocated from the same NUMA node with `restricted` or `single-numa-node` Topology Manager polices.
```
Comment (jboxman) [inline modules/nw-sriov-topology-manager.adoc]: What's NUMA mean?

Comment (lamek) [reply modules/nw-sriov-topology-manager.adoc]: I'll confirm with the team, but I think it's non-uniform memory access. And I'll spell it out here either way.

File: modules/nw-sriov-topology-manager.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+. Optional: Confirm that the `sample-pod` is allocated with exclusive CPUs.
++
+----
+oc exec sample-pod -- cat /sys/fs/cgroup/cpuset/cpuset.cpus
+----
+
+. Optional: Confirm that the SR-IOV device and CPUs that are allocated for the `sample-pod` are on the same NUMA node.
++
+----
+oc exec sample-pod -- cat /sys/fs/cgroup/cpuset/cpuset.cpus
```
Comment (zhaozhanqi) [inline modules/nw-sriov-topology-manager.adoc]: same command with above step?

**Pull request merged**